js/views/modals/purchase/Moderators.js

```diff
@@ -184,7 +184,7 @@ export default class extends baseVw {
       this.removeNotFetched(model.id);
       this.modCards.push(newModView);
       // if required, select the first  moderator
-      if (this.options.selectFirst && !this.firstSelected) {
+      if (this.options.selectFirst && !this.firstSelected && !this.noneSelected) {
```

Comment:
The noneSelected can be passed in to prevent the first moderator from being selected when the total is too low.